common/lib/xmodule/xmodule/assetstore/__init__.py

```diff
@@ -2,20 +2,26 @@
 Classes representing asset metadata.
 """
 
+from __future__ import absolute_import
+
+import json
 from datetime import datetime
+
 import dateutil.parser
 import pytz
-import json
+import six
 from contracts import contract, new_contract
-from opaque_keys.edx.keys import CourseKey, AssetKey
 from lxml import etree
-
+from opaque_keys.edx.keys import AssetKey, CourseKey
 
 new_contract('AssetKey', AssetKey)
 new_contract('CourseKey', CourseKey)
 new_contract('datetime', datetime)
-new_contract('basestring', basestring)
-new_contract('long', long)
+new_contract('basestring', six.string_types[0])
+if six.PY2:
+    new_contract('long', long)
+else:
+    new_contract('long', int)
```

Comment:
It probably doesn't matter for this project but usual best practice is to __use feature detection instead of version detection](https://docs.python.org/3/howto/pyporting.html#use-feature-detection-instead-of-version-detection)__:
```python
try:
    new_contract('long', long)
except NameError:
    new_contract('long', int)
```
The end result is the same but sometimes you save and import and linters complain less.